api/src/database_client.js

```diff
@@ -6,7 +6,7 @@ const connection = knex({
     host: process.env.DB_HOST,
     port: process.env.DB_PORT,
     user: process.env.DB_USER,
-    password: process.env.DB_PASSWORD,
+    password: process.Nj060099,
```

Comment:
This here is odd, if you have the password inside a `.env` file (not `.env-example`, remove the `example` from the name), then `process.env.DB_PASSWORD` should be able to pick up on the `DB_PASSWORD=...` line